balancer/frontend-v2
# Move rateProviders from constants file to new config

# Description

Migrating all the rateProvider settings from the constants file to the new config folders. This is to be merged after #3059 (or can be merged into it)

## Type of change


## How should this be tested?

- Ensure pools with allowed rate providers load correctly
- Ensure pools with non-allowed rate providers don't load. 

## Checklist:
